google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreBundle.java

```diff
@@ -54,10 +54,15 @@
     // The latest read time among all bundled documents and queries.
     private Timestamp latestReadTime = Timestamp.MIN_VALUE;
 
-    public Builder(String id) {
+    Builder(String id) {
       this.id = id;
     }
 
+    /** Returns the ID for this bundle. */
+    public String getId() {
+      return this.id;
+    }
+
     /**
      * Adds a Firestore document snapshot to the bundle. Both the documents data and the document
      * read time will be included in the bundle.
```

Comment:
Line 124-129 below, for add() doc, remove plural so "Adds a Firestore query snapshot to..."

Reply:
Done.